pkg/querier/store_gateway_client.go

```diff
@@ -58,17 +57,7 @@ func (c *storeGatewayClient) String() string {
 	return c.conn.Target()
 }
 
-func newStoreGatewayClientPool(discovery client.PoolServiceDiscovery, logger log.Logger, reg prometheus.Registerer) *client.Pool {
-	// We prefer sane defaults instead of exposing further config options.
-	clientCfg := grpcclient.Config{
```

Comment:
Unfortunately having used `grpcclient.Config` now we have added more complexity to the config for something we don't really want to let configure at this stage.

Reply:
These can practically be left un-configured because they will continue to use the defaults.

Reply:
In the ruler you just imported the TLS config. Can we do the same thing here, please?